frontend/src/features/documents/documentUpload/DocumentUploadView.test.tsx

```diff
@@ -0,0 +1,169 @@
+import { createMemoryHistory } from 'history';
+import { mockLookups } from 'mocks/mockLookups';
+import { Api_DocumentType } from 'models/api/Document';
+import { Api_Storage_DocumentTypeMetadataType } from 'models/api/DocumentStorage';
+import { lookupCodesSlice } from 'store/slices/lookupCodes';
+import { act, fireEvent, render, RenderOptions, userEvent, waitFor } from 'utils/test-utils';
+
+import { DocumentMetadataForm } from '../ComposedDocument';
+import DocumentUploadView from './DocumentUploadView';
+
+const history = createMemoryHistory();
+
+const handleSubmit = jest.fn();
+
+const handleCancelClick = jest.fn();
+const onUploadDocument = jest.fn();
+const onDocumentTypeChange = jest.fn();
+const documentTypes: Api_DocumentType[] = [
+  {
+    id: 1,
+    documentType: 'BC Assessment Search',
+    mayanId: 17,
+  },
+  {
+    id: 2,
+    documentType: 'Privy Council',
+    mayanId: 7,
+  },
+];
+const documentTypeMetadata: Api_Storage_DocumentTypeMetadataType[] = [
+  {
+    id: 1,
+    document_type: {
+      id: 1,
+      label: 'BC Assessment Search',
+    },
+    metadata_type: {
+      default: '',
+      id: 1,
+      label: 'Tag',
+      lookup: '',
+      name: 'Tag',
+      parser: '',
+      parser_arguments: '',
+      url: '',
+      validation: '',
+      validation_arguments: '',
+    },
+    required: true,
+  },
```

Comment:
again, these could be reused if they were moved to shared **mocks** folder

Reply:
done